GitUI/UserControls/BlameControl.cs

```diff
@@ -388,6 +435,48 @@ private void BuildAuthorLine(GitBlameLine line, StringBuilder lineBuilder, strin
             }
         }
 
+        private IList<Color> GetGradients()
+        {
+            // Color chosen from: https://colorbrewer2.org/#type=sequential&scheme=Greens&n=9
+            return new List<Color>
+            {
+                Color.FromArgb(247, 252, 245).AdaptBackColor(),
+                Color.FromArgb(229, 245, 224).AdaptBackColor(),
+                Color.FromArgb(199, 233, 192).AdaptBackColor(),
+                Color.FromArgb(161, 217, 155).AdaptBackColor(),
+                Color.FromArgb(116, 196, 118).AdaptBackColor(),
+                Color.FromArgb(65, 171, 93).AdaptBackColor(),
+                Color.FromArgb(35, 139, 69).AdaptBackColor(),
+                Color.FromArgb(0, 109, 44).AdaptBackColor(),
+                Color.FromArgb(0, 68, 27).AdaptBackColor(),
+            };
```

Comment:
A purely stylistic suggestion: we could avoid repeating AdaptBackColor() by using `.Select()`

```csharp
new [] 
{
    Color.FromArgb(247, 252, 245),
    // ...
}.Select(ColorHelper.AdaptBackColor).ToList()
```